### ISSUE TITLE
[BUG][TGL]The Dmic  volume controls were renamed from "Dmic0 Capture Volume" to "Dmic0 10 Capture Volume"

### ISSUE THREAD

**Issue by Liviali155** [thesofproject/sof#3056]
**Describe the bug**
The Dmic  volume controls were renamed from "Dmic0 Capture Volume" to "Dmic0 10 Capture Volume"

**To Reproduce**
1."sudo reboot " to reboot sysytem
2.amixer contents |less" to check the name of  Dmic volume controls

**Reproduction Rate**
10/10

**Expected behavior**
The name of Dmic  volume controls is  "Dmic0 Capture Volume"

**Impact**
The name of Dmic  volume controls is  "Dmic0 10 Capture Volume"
When adjust  Input volume in sound setting ,Dmic0 10 Capture Volume in amixer contents will not change,after applied #2942,The name of Dmic  volume controls is  "Dmic0 Capture Volume",when adjust input volume in sound setting, "Dmic0 Capture Volume" will change  at the same time

```
numid=51,iface=MIXER,name='Dmic0 10 Capture Switch'
  ; type=BOOLEAN,access=rw------,values=2
  : values=on,on
numid=50,iface=MIXER,name='Dmic0 10 Capture Volume'
  ; type=INTEGER,access=rw---R--,values=2,min=0,max=70,step=0
  : values=70,70
  | dBscale-min=-50.00dB,step=1.00dB,mute=1
numid=53,iface=MIXER,name='Dmic1 Capture Volume'
  ; type=INTEGER,access=rw---R--,values=2,min=0,max=70,step=0
  : values=50,50
```

**Environment**
1) Branch name and commit hash of the 2 repositories: sof (firmware/topology) and linux (kernel driver).
    * Kernel: {sof-dev:8b7c1b80}
    * SOF: {master:2afbbd0e}
2) Name of the topology file
    * Topology: sof-hda-generic-2ch.tplg
3) Name of the platform(s) on which the bug is observed.
    * Platform: TGL laptop with aboard codec ALC3204  in HDA mode

[dmesg0616.log](https://github.com/thesofproject/sof/files/4785582/dmesg0616.log)
[sof-logger0616.log](https://github.com/thesofproject/sof/files/4785583/sof-logger0616.log)

**Comment by Liviali155** [comment]
Need to merge https://github.com/thesofproject/sof/pull/2942

**Comment by Liviali155** [comment]
closed  for #2942 merged
